content/tutorial-paychan.md

```diff
@@ -0,0 +1,489 @@
+# Payment Channels Tutorial
+
+Payment Channels are an advanced feature for sending "asynchronous" XRP payments that can be divided into very small increments and settled later. This tutorial walks through the entire process of using a payment channel, with examples using the [JSON-RPC API](reference-rippled.html) of a local `rippled` server.
+
+1. **The payer creates a payment channel to a particular recipient.** In doing so, the payer pre-funds the channel with XRP.
+
+    This is a [PaymentChannelCreate transaction][]. As part of this process, the payer sets certain specifics of the channel like an expiration time and a settlement delay, which affect the guarantees around the claims in the channel. The payer also sets the public key that will be used to verify claims against the channel.
+
+    **Tip:** The "settlement delay" does not delay the settlement, which can happen as fast as a ledger version closes (3-5 seconds). The "settlement delay" is a forced delay on closing the channel so that the payee has a chance to finish with settlement.
+
+    The following example shows creation of a payment channel by submitting to a local `rippled` server with the JSON-RPC API. The payment channel allocates 100 XRP from rN7n7... to rf1Bi... with a settlement delay of 1 day. The public key happens to be rN7n7...fzRH's master public key.
+
+    Request:
+
+        POST http://localhost:5005/
+        Content-Type: application/json
+
+        {
+            "method": "submit",
+            "params": [{
+                "secret": "s████████████████████████████",
+                "tx_json": {
+                    "Account": "rN7n7otQDd6FczFgLdSqtcsAUxDkw6fzRH",
+                    "TransactionType": "PaymentChannelCreate",
+                    "Amount": "100000000",
+                    "Destination": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
+                    "SettleDelay": 86400,
+                    "PublicKey": "023693F15967AE357D0327974AD46FE3C127113B1110D6044FD41E723689F81CC6",
+                    "DestinationTag": 20170428
+                },
+                "fee_mult_max": 1000
+            }]
+        }
+
+    Response:
+
+        200 OK
+
+        {
+            "result": {
+                "engine_result": "tesSUCCESS",
+                "engine_result_code": 0,
+                "engine_result_message": "The transaction was applied. Only final in a validated ledger.",
+                ...
+                "tx_json": {
+                    ...
+                    "TransactionType": "PaymentChannelCreate",
+                    "hash": "3F93C482C0BC2A1387D9E67DF60BECBB76CC2160AE98522C77AF0074D548F67D"
+                }
+            }
+        }
+
+    The payer should check the transaction's final result in a validated ledger and get the Channel ID from the metadata. This can be done with the `tx` command:
```

Comment:
I'd rather not make every "submit transaction; now check the results" into multiple sub-steps. That's better covered in detail by the Reliable Transaction Submission document.